### DISCUSSION TITLE
I tried to enter the community from https://t.me/Avalonia, but encountered "this group can't be displayed." What should I do?

### DISCUSSION THREAD

**Discussion by Linlccc** [AvaloniaUI/Avalonia#17626] [General]
I tried to enter the community from https://t.me/Avalonia, but encountered "this group can't be displayed." What should I do?

**Comment by maxkatz6** [comment]
Could there be a regional issue? This chat is still active.
![image](https://github.com/user-attachments/assets/ac8545a7-d9e8-4fd4-aa1d-d9b17be2cc56)

**Reply by Linlccc** [reply]
It should be a network problem. It's normal for me to stay at home.
